resources/ecr-scan-check.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python3
+
+import argparse
+import boto3
+import json
+import os
+import sys
+import time
+
+
+def logoutput(level, message):
+    colours = {
+        "Info": "\033[1;34m",
+        "Error": "\033[0;31m",
+        "Success": "\033[0;32m",
+        "end": "\033[0m",
+    }
+
+    string = "{}{}:{} {}".format(colours[level], level, colours["end"], message)
+    print(string)
+
+
+def get_severities(min_severity):
+    severities = ["CRITICAL", "HIGH", "MEDIUM", "LOW", "INFORMATIONAL"]
+
+    if min_severity == "ANY":
+        selected_severities = severities
+    else:
+        selected_severities = []
+        for severity in severities:
+            selected_severities.append(severity)
+            if severity == min_severity:
+                break
+
+    return selected_severities
+
+
+def ecr_open_session():
+    logoutput("Info", "Logging in to ECR")
+    try:
+        session = boto3.Session(
+            aws_access_key_id=os.environ.get("AWS_ACCESS_KEY_ID"),
+            aws_secret_access_key=os.environ.get("AWS_SECRET_ACCESS_KEY"),
+            region_name=os.environ.get("AWS_REGION"),
+        )
+        ecr_client = session.client("ecr")
+
+    except session.exceptions.UnauthorizedException as err:
+        logoutput("Error", "Unable to authorise request. Invalid credentials")
```

Comment:
If this is thrown due to authorisation then this message may be a little misleading. I.e. we may need to differentiate better between authentication and authorisation